build.gradle

```diff
@@ -29,7 +34,11 @@ subprojects {
   }
 
   dependencies {
-    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-spring', version: '1.5.0'
+    compile group: 'uk.gov.hmcts.reform', name: 'java-logging-appinsights', version: versions.reformLogging
```

Comment:
Don't we need instrumentation key config if we enable app insights?

Reply:
Also `applicationinsights-agent-2.0.0-BETA.jar` needs to be in lib ?

Reply:
as long as it's not testing in master we can test in the demo branch?

Reply:
as long as integration tests pass and app boots up - addition of this library is considered a success 🏆